docs/spec/resolution.md

```diff
@@ -108,15 +108,16 @@ If the compiled DID ****was not**** determined to be _Unresolvable_, as defined
 
 1. Generate a JSON object for the _Resolution Result_, structured in accordance with the [Decentralized Identifier Resolution](https://w3c-ccg.github.io/did-resolution/#example-14-example-did-resolution-result) specification.
 2. Set the `didDocument` property of the _Resolution Result_ object to the resolved DID Document generated via the [Operation Compilation](#operation-compilation) process.
-5. The _Resolution Result_ object ****MUST**** include a `didDocumentMetadata` property, and its value is an object composed of the following values:
-    - `canonicalId` - If canonical representation of the resolved DID exists, the presence of canonical DID representation within the `didDocumentMetadata` is determined as follows:
+5. The _Resolution Result_ object ****MUST**** include a `didDocumentMetadata` property, and its value ****MUST**** be an object composed of the following properties:
+    - `deactivated` - This property ****MUST**** be present if the resolved DID is determined to be in a deactivated state, and it ****MUST**** be set to the boolean value `true`. If the resolved DID is not in a deactivated state, this value ****MUST**** be set to the boolean value `false`.
```

Comment:
Issue #1015 filed to track pending implementation to align with changes in this section.